examples/example-gateway/build/endpoints/tchannel/baz/baz_simpleservice_method_call_tchannel.go

```diff
@@ -150,6 +158,51 @@ func (h *SimpleServiceCallHandler) Handle(
 	return err == nil, &res, resHeaders, nil
 }
 
+// redirectToDeputy sends the request to deputy hostPort
+func (h *SimpleServiceCallHandler) redirectToDeputy(
+	ctx context.Context,
+	reqHeaders map[string]string,
+	hostPort string,
+	req *endpointsTchannelBazBaz.SimpleService_Call_Args,
+	res *endpointsTchannelBazBaz.SimpleService_Call_Result,
+) (bool, zanzibar.RWTStruct, map[string]string, error) {
+	var routingKey string
+	if h.Deps.Default.Config.ContainsKey("tchannel.routingKey") {
+		routingKey = h.Deps.Default.Config.MustGetString("tchannel.routingKey")
+	}
+
+	serviceName := h.Deps.Default.Config.MustGetString("tchannel.serviceName")
+	timeout := time.Millisecond * time.Duration(
+		h.Deps.Default.Config.MustGetInt("tchannel.deputy.timeout"),
+	)
+
+	timeoutPerAttempt := time.Millisecond * time.Duration(
+		h.Deps.Default.Config.MustGetInt("tchannel.deputy.timeoutPerAttempt"),
+	)
+
+	methodNames := map[string]string{
+		"SimpleService::Call": "Call",
+	}
+
+	h.Deps.Default.Channel.GetSubChannel(serviceName, tchannel.Isolated)
```

Comment:
the subchannel here has a peer list that is designated for the subchannel, we should just add the hostport as peer to the peer list.

Reply:
Once the peer is added to peer list to the subchannel, if the subchannel.beginCall, it would use round robin to select peer (prod and deputy) based on the peer score to make a outbound call. Hence the prod traffic has a possibility to send request to deputy.

Reply:
For an isolated subchannel, the round robin is among the subchannel's peerlist, not top channel's peerlist.

Reply:
Yes, I confirm it is round robin around subchannel's peerlist.